doc/daal4py/model-builders.rst

```diff
@@ -19,64 +19,69 @@
-LightGBM*, and CatBoost* frameworks.
+The most popular implementations of it are: 
+
+* XGBoost*
+* LightGBM*
+* CatBoost*
+
 daal4py Model Builders deliver the accelerated
 models inference of those frameworks. The inference is performed by the oneDAL GBT implementation tuned 
 for the best performance on the Intel(R) Architecture. 
 
 Conversion
 ---------
-The first step is to convert already trained model. There are similar 
-APIs for different frameworks. 
+The first step is to convert already trained model. The
+API usage for different frameworks is the same:
+
 XGBoost::
 
   import daal4py as d4p
-  d4p_model = d4p.get_gbt_model_from_xgboost(xgb_model)
+  d4p_model = d4p.mb.convert_model(xgb_model)
 
 LightGBM::
 
   import daal4py as d4p
-  d4p_model = d4p.get_gbt_model_from_lightgbm(lgb_model)
+  d4p_model = d4p.mb.convert_model(lgb_model)
 
 CatBoost::
 
   import daal4py as d4p
-  d4p_model = d4p.get_gbt_model_from_catboost(cb_model)
+  d4p_model = d4p.mb.convert_model(cb_model)
 
 .. note:: Convert model only once and then use it for the inference.
 
 Classification and Regression Inference
----------
-GBT implementation in daal4py assumes separate APIs for the classification and regression.
-Specify the corresponding API and match the corresponding problem 
-in the initial framework.
-
-Classification::
+----------------------------------------
 
-    d4p_cls_algo = d4p.gbt_classification_prediction(
-        nClasses=params['classes_count'],
-        resultsToEvaluate="computeClassLabels",
-        fptype='float'
-    )
+The API is the same for classification and regression inference. 
+Based on the original model passed to the ``convert_model``, ``d4p_prediction`` is either the classification or regression output. 
+    
+    ::
+      
+      daal_prediction = daal_model.predict(test_data)
 
-Regression::
-    d4p_reg_algo = d4p.gbt_regression_prediction()
+Here, the ``predict()`` method of ``daal_model`` is being used to make predictions on the ``test_data`` dataset.
+The ``daal_prediction``variable stores the predictions made by the ``predict()`` method. 
 
-Next, daal4py algorithm object needs compute method to be called. 
-Both the data and the previously converted model should be passed with the results of the prediction 
-available within the  ``.prediction`` parameter.
+Scikit-learn-style Estimators
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Compute::
+You can also use the scikit-learn-style classes ``GBTDAALClassifier`` and ``GBTDAALRegressor`` to convert and infer your models. For example:
 
-    d4p_predictions = d4p_reg_algo.compute(X_test, d4p_model).prediction
+:: 
 
-The one-line variant of the same code::
-    d4p_prediction = d4p.gbt_regression_prediction().compute(X_test, d4p_model).prediction
+  from daal4py.sklearn.ensemble import GBTDAALRegressor
+  reg = xgb.XGBRegressor()
+  reg.fit(X, y)
+  d4p_predt = GBTDAALRegressor.convert_model(reg).predict(X)
 
 
 Limitations
```

Comment:
Let's also remove this limitation section. Missing values support was added as well and section is no longer relevant